cmd/tls_integration_test.go

```diff
@@ -0,0 +1,67 @@
+//go:build !windows
+// +build !windows
+
+package cmd
+
+// TODO: convert this into the integration tests, once https://github.com/grafana/k6/issues/2459 will be done
+
+import (
+	"encoding/json"
+	"path/filepath"
+	"testing"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type metric struct {
+	Avg float64 `json:"avg"`
+	Min float64 `json:"min"`
+	Max float64 `json:"max"`
+}
+
+type summaryMetrics struct {
+	HTTPReqTLSHandshaking metric `json:"http_req_tls_handshaking"`
+}
+
+type summaryMetricsWrapper struct {
+	Metrics summaryMetrics `json:"metrics"`
+}
+
+func TestTLSLoadSystemCertificates(t *testing.T) {
+	t.Parallel()
+
+	script := `
+		import http from "k6/http";
+
+		export default function() {
+			let response = http.get("https://test-api.k6.io");
+		};
+	`
+
+	testFilename := "script.js"
+	summaryFilename := "summary.json"
+	testState := newGlobalTestState(t)
+	require.NoError(t, afero.WriteFile(testState.fs, filepath.Join(testState.cwd, testFilename), []byte(script), 0o644))
+	testState.args = []string{"k6", "run", testFilename, "--summary-export", summaryFilename, "-i", "2"}
```

Comment:
```suggestion
	testState.args = []string{"k6", "run", "--no-connection-reuse", "--summary-export", summaryFilename, "-i", "2", testFilename}
```
You're missing the option to re-init the connection, this is the reason why you get one request with a 0 value. If you change it, you will see the min value increase as expected.